usvm-jvm/src/main/kotlin/org/usvm/machine/JcApproximations.kt

```diff
@@ -89,6 +77,12 @@ class JcMethodApproximationResolver(
     private val usvmApiSymbolicMap by lazy { ctx.cp.findClassOrNull<SymbolicMap<*, *>>() }
     private val usvmApiSymbolicIdentityMap by lazy { ctx.cp.findClassOrNull<SymbolicIdentityMap<*, *>>() }
 
+    @Suppress("RecursivePropertyAccessor")
```

Comment:
This appears to be an Intellij warning, not a Kotlin compiler warning. Please, don't suppress such warnings.